src/java.base/share/classes/sun/security/x509/AlgorithmId.java

```diff
@@ -203,7 +203,13 @@ public void derEncode (OutputStream out) throws IOException {
                 bytes.putNull();
             }*/
             if (algid.equals(RSASSA_PSS_oid) || algid.equals(ed448_oid)
-                    || algid.equals(ed25519_oid)) {
+                    || algid.equals(ed25519_oid)
+                    || algid.equals(x448_oid)
+                    || algid.equals(x25519_oid)
+                    || algid.equals(SHA224withECDSA_oid)
+                    || algid.equals(SHA256withECDSA_oid)
+                    || algid.equals(SHA384withECDSA_oid)
+                    || algid.equals(SHA512withECDSA_oid)) {
                 // RFC 4055 3.3: when an RSASSA-PSS key does not require
                 // parameter validation, field is absent.
```

Comment:
Would you like to add more comments on other RFCs here?

Reply:
Added the comments for RFC reference.